### PR TITLE
chore: add beumer group to USERS

### PR DESCRIPTION
**Pull request by th0ger**
Update USERS. @syayi

### CODE REVIEW

**Review by yhl25 — APPROVED** [review]

File: USERS.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ Please add your company name and initial use case (optional) below.
 4. [Valegachain Analytics](https://www.valegachain.com/) Numaflow is used to extract, transform, and load cryptocurrency blocks and mempool transactions in data lakes, as well as for activity alerts.
 5. [Lockheed Martin](https://lockheedmartin.com/) Perform ELT processing on high and low volume data streams of sensor data as recieved from IOT type systems.
 6. [Seekr](https://www.seekr.com/) Numaflow coordinates multiple ML pipelines to rate and extract information from the pipeline input.
+7. [BEUMER Group](https://https://www.beumergroup.com/) - Signal processing for IoT-sensors, providing predictive maintenance solutions for the Intra-logistics sector.
```
Comment (yhl25) [inline USERS.md:11]:
```suggestion
7. [BEUMER Group](https://www.beumergroup.com/) - Signal processing for IoT-sensors, providing predictive maintenance solutions for the Intra-logistics sector.
```

**Comment by th0ger** [comment]
Will fix the http:// typo.

File: USERS.md (hunk 1 above)
Comment (th0ger) [reply USERS.md]: fixed

**Review by whynowy — APPROVED** [review]
Thanks!

**Pull request merged**